internal/orchestrator/app/app_test.go

```diff
@@ -26,26 +26,26 @@ import (
 
 func TestLoad(t *testing.T) {
 	t.Run("it fails if the app path is empty", func(t *testing.T) {
-		app, err := Load("")
+		app, err := Load(nil)
```

Comment:
maybe this is more correct?
```suggestion
		app, err := Load(path.New(""))
```